### ISSUE TITLE
Avg duration 0ms

### ISSUE THREAD

**Issue by rus-ik** [darold/pgbadger#676]
In Most Frequent Queries, if Min duration is eq to 0, then number in Avg is zero too (and it seems it's not correct):
![image](https://user-images.githubusercontent.com/743185/127076149-bc9f92c8-fa67-4066-bebe-095d269b22df.png)

**Comment by wsdookadr** [comment]
@rus-ik Since The avg duration is expressed as an rounded number of milliseconds. Let's take the last row, I think the average should be the following: `(15000 + 696) / 300281 = 0.0522` which is rounded to zero.

**Comment by rus-ik** [comment]
It sounds correct, thanks.. but I'd rather suggest to use a percentile (or a quantile) in this case, since the avg (when it goes below zero) becomes useless and shows nothing....

**Comment by darold** [comment]
I agree that having percentile reported would be useful, but may increase the time to generate reports. Must be evaluated.
